### DISCUSSION TITLE
translateable notification

### DISCUSSION THREAD

**Discussion by mateusgalasso** [laravel/nova-issues#4153] [Ideas & Feature Requests]
Could you please make the notification part translateable
![image](https://user-images.githubusercontent.com/3990028/166510804-10a5fb18-327b-411c-bc37-5323358807fc.png)

Would be greate
Thanks

**Comment by davidhemphill** [comment]
Which part is not translatable?

**Reply by mateusgalasso** [reply]
The part in english
"NOTIFICATIONS" and "Mark all as Read"
at the code is fixed, not translatable


NotificationCenter.vue
```
<Heading :level="3">Notifications</Heading>
          <LinkButton @click="markAllNotificationsAsRead" class="ml-auto"
            >Mark all as Read
          </LinkButton>
```

**Reply by davidhemphill** [reply]
Oops! Sorry about that. Just pushed a release fixing this.
